### PR TITLE
Update readme to reflect SQLite nuances 

### PR DESCRIPTION
**Pull request by joshuabamboo**
Tried to beginnerify™  the explanation of datatypes in SQLite while giving an accurate representation of how SQLite stores data. Also this should close issue #5. 

Would love to get input on the NUMERIC type affinity. It is unclear to me how to describe that in a simple, clear way.

:eyes: @SophieDeBenedetto @AnnJohn @danny-sullivan @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
...
 ### TEXT
 
 Any alphanumeric characters which we want to represent as plain text. The body of this paragraph is text. Your name is text. Your email address is a piece of text. Your height, weight, and age, however, are probably not.
 
+Common text datatypes include:
+
+```bash
+	CHARACTER(20)
+	VARCHAR(225)
+	TEXT
```
Comment (jmburges) [inline README.md]: I would have an opinion on which of these is the most useful. In general we just use `TEXT`.

File: README.md [hunk 2]
```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
...
+```
+
 ### INTEGER
 
 Anything we want to represent as a whole number. If it's a number and contains no letter or special characters or decimal points then we should store it as an integer. If we use it to perform math or create a comparison between two different rows in our database, then we definitely want to store it as an integer. If it's just numbers, it's generally not a bad idea to store it as an integer. You might never add two house address numbers together, but you might want to sort them numerically. For example in the preceding case, you might want to get the biggest number and not the longest piece of text.
 
+Common integer datatypes include:
+
+```bash
+	INT
```
Comment (jmburges) [inline README.md]: One liner explanation of the diff, and then have an opinion. Which one should we use? Whenever we give options, we should give opinions

File: README.md [hunk 3]
```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
...
 ### REAL
 
 Anything that's a plain old decimal like 1.3 or 2.25. SQLite will store decimals up to 15 characters long. You can store 1.2345678912345 or 1234.5678912345, but 1.23456789123456789 would only store 1.2345678912345. In other database systems this is called 'double precision.'
 
 With these three types in hand, we are going to be able to work our way through the next several topics, and this whole typing concept is going to quickly be second nature for you.
+
+Common real datatypes include:
+
+```bash
+	FLOAT
```
Comment (jmburges) [inline README.md]: Same dealio

File: README.md [hunk 4]
```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
...
+	REAL
+```
+
+###NUMERIC
+
+Common numeric datatypes include:
```
Comment (jmburges) [inline README.md]: Have an opinion on this. I think the answer is the only useful ones here are BOOLEAN and DATETIME

**Comment by jmburges** [comment]
few comments!

**Comment by joshuabamboo** [comment]
@jmburges I updated it to remove all the mentions of other datatypes. SQLite really only has four datatypes that are technically "storage classes". Every other datatype is simply coerced into one of the four storage classes.

**Comment by joshuabamboo** [comment]
:shipit:? @jmburges

**Comment by jmburges** [comment]
:shipit:

**Pull request merged**